src/main/java/com/google/alpollo/model/SongEntity.java

```diff
@@ -9,12 +11,23 @@
  */
 public class SongEntity {
   private final String name;
-  // Each entity has a salience score, telling us how important the word is in the given context.
+  /**  Each entity has a salience score, telling us how important the word is in the given context. */
   private final double salience;
+  /** 
+   * The API sets each entity to a category.
+   * e.g. PERSON, LOCATION, WORK_OF_ART*/ 
+  private final String type;
+  /** 
+   * For most entity types, the metadata is a Wikipedia URL. Other times, it can be extra info.
+   * An object containing a list of "key": value pairs. Example: { "name": "wrench", "mass": "1.3kg", "count": "3" }.
+  */
+  private final Map<String, String> metadata;
```

Comment:
I don't think that we need all metadata, because we have only an idea of how to use `Wikipedia URL` now and not smth else.

Reply:
I'm not sure we can remove stuff from the Map, I'll reasearch a bit.

Reply:
you just get map with metadata using `entity.getMetadataMap()` and get the value of the key `wikipedia_url` (or how it's called) and that's all, if this key doesn't exist, then you can set it to null or smth like that, as you will decide